java-accessapproval/versions.txt

```diff
@@ -1,6 +1,6 @@
 # Format:
 # module:released-version:current-version
 
-google-cloud-accessapproval:2.4.8:2.4.9-SNAPSHOT
-grpc-google-cloud-accessapproval-v1:2.4.8:2.4.9-SNAPSHOT
-proto-google-cloud-accessapproval-v1:2.4.8:2.4.9-SNAPSHOT
+google-cloud-accessapproval:2.4.8:2.4.8
```

Comment:
We have not yet released so having "-SNAPSHOT" is correct, isn't it?

Reply:
Hasn't it been released? https://search.maven.org/artifact/com.google.cloud/google-cloud-accessapproval/2.4.8/jar